core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/CoarseMesosSchedulerBackend.scala

```diff
@@ -101,6 +101,10 @@ private[spark] class CoarseMesosSchedulerBackend(
   private val slaveOfferConstraints =
     parseConstraintString(sc.conf.get("spark.mesos.constraints", ""))
 
+  // reject offers with mismatched constraints in seconds
+  private val rejectOfferDurationForUnmetConstraints =
+    sc.conf.getDouble("spark.mesos.rejectOfferDurationForUnmetConstraints", 120)
```

Comment:
please use `conf.getTimeAsSeconds` instead, and maybe I would call this `spark.mesos.unmetConstraintsRejectOfferDuration`